validphys2/src/validphys/checks.py

```diff
@@ -52,6 +52,11 @@ def check_pdf_is_montecarlo_or_hessian(pdf, **kwargs):
     )
 
 
+@make_argcheck
+def check_not_using_pdferr(use_pdferr, **kwargs):
+    check(not use_pdferr, "The flag 'use_pdferr' must be off to use this function")
```

Comment:
```suggestion
    check(not use_pdferr, "The flag 'use_pdferr' must be `False` to use this function. This is to avoid including the PDF error in the uncertainty bars of the experimental datapoints.")
```
Or something like that, anyway I think a bit more explanation would be helpful